internal/services/sentinel/sentinel_data_connector_microsoft_threat_intelligence.go

```diff
@@ -56,22 +57,27 @@ func (s DataConnectorMicrosoftThreatIntelligenceResource) Arguments() map[string
 			ValidateFunc: validation.IsUUID,
 		},
 
-		"bing_safety_phishing_url_lookback_date": {
+		"microsoft_emerging_threat_feed_lookback_date": {
 			Type:         pluginsdk.TypeString,
-			Optional:     true,
 			ForceNew:     true,
+			Required:     true,
```

Comment:
As this is a breaking change, let's set Required/Optional by the `FourPointOh` feature flag. We also need to update the docs to say this is Required now